fastlane/Fastfile

```diff
@@ -34,7 +34,7 @@ ALL_BUNDLE_IDENTIFIERS = [
   'org.wordpress.WordPressAllTimeWidget',
   'org.wordpress.WordPressThisWeekWidget',
   'org.wordpress.WordPressIntents'
-].freeze
+]
```

Comment:
Why don't we want to `.freeze` this? I can't imagine we want it mutable? (not blocking)

Reply:
We don't, but `match` does 😭 

Under the hood, it calls `.flatten!` on the value which [crashes](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-iOS/22013/workflows/b25664b2-3ce6-49f4-a795-818a9673f67e/jobs/50408) if this is a constant.

See also:

- fastlane/fastlane#11324
- fastlane/fastlane@b817148/match/lib/match/runner.rb#L84-L86